cucumber/authenticators_jwt/features/authn_jwt_input_validation.feature

```diff
@@ -0,0 +1,108 @@
+Feature: JWT Authenticator - Input Validation
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !group users
```

Comment:
The original file was hosts. Do we want to change this?

Reply:
In this specific yes because we don't use token and want to control identity from url. Anyway we don't even come to this step because these are tests for failiure in input validation